autogen/main/variables.tf.tmpl

```diff
@@ -889,3 +889,11 @@ variable "fleet_project_grant_service_agent" {
   default = false
 }
 {% endif %}
+{% if beta_cluster and autopilot_cluster %}
+
+variable "logging_variant" {
+  description = "(Optional) The type of logging agent that is deployed by default for newly created node pools in the cluster. Valid values include DEFAULT and MAX_THROUGHPUT."
+  type        = string
+  default     = "DEFAULT"
```

Comment:
As this will override the cluster's value, we should probably default to `null`.  This will also avoid config churn on initial apply.

Reply:
Good point -- I have changed the default to `null`